packages/vanilla/src/complex/array/ArrayControl.tsx

```diff
@@ -44,38 +44,41 @@ export const ArrayControl = ({
     () => findUISchema(uischemas, schema, uischema.scope, path, undefined, uischema, rootSchema),
     [uischemas, schema, uischema.scope, path, uischema, rootSchema]
   );
+  const isValid = errors.length === 0;
+  const divClassNames = [validationClass]
```

Comment:
`validationClass` does not exist yet. Should probably look like this:

```
  const validationClass = getStyleAsClassName('array.control.validation');
```

`getStyleAsClassName` can also be added to the deconstructor